examples/reference/elements/bokeh/Bars.ipynb

```diff
@@ -158,16 +181,29 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For full documentation and the available style and plot options, use ``hv.help(hv.Bars).``"
+    "For full documentation and the available style and plot options, use ``hv.help(hv.Bars)``."
    ]
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
   "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
```

Comment:
Please strip out the metadata, using something like `strip_kernel`:

```
#!/bin/bash
cp "$1" /tmp/
/usr/local/bin/jq --indent 1 \
    '
    (.cells[] | select(has("outputs")) | .outputs) = []
    | (.cells[] | select(has("execution_count")) | .execution_count) = null
    | .metadata = {"language_info": {"name":"python", "pygments_lexer": "ipython3"}}
    | .cells[].metadata = {}
    ' "$1" > /tmp/stripped.ipynb
cat /tmp/stripped.ipynb > "$1"
```

Reply:
Your snippet doesn't actually address the `nbformat`. Should both of those be removed completely?

Reply:
That's fine to leave; it needs to have some value, and whether that's 4.2 or 4.4 shouldn't matter.